cmr_images/ReadMe.md

```diff
@@ -0,0 +1,25 @@
+# Cardiac MR Image (Short-Axis) Dataset for Pulmonary Arterial Hypertension Diagnosis
+
+This dataset consists of the CMR images acquired from 179 subjects. The images have been processed in size (64x64) and (32x32). The images are shared in DICOM format.
+The files are organized in the following structure:
+
+|-- DICOM \
+|--|-- Subject_1 \
+|--|--|-- Phase_1.dcm \
+|--|--|-- Phase_2.dcm \
+|--|--|-- ... \
+|--|--|-- Phase_20.dcm \
+|--|-- Subject_2 \
+|--|-- Subject_3 \
+|--|-- … \
+|--|-- Subject_179 \
+|-- Mask \
+|--|-- Mask_WxH \
+|--|--|-- Phase_1.dcm \
+|-- Landmarks_WxH.csv
+
+W, H denotes width and height of processed images, e.g. 64 or 32.
+
+## Reference
```

Comment:
Ask users to cite us explicitly like in http://www.vision.caltech.edu/Image_Datasets/Caltech101/